src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const objKeys = Object.keys(robot);
+  const objValues = Object.values(robot);
+  const trueRobot = {};
+
+  for (let i = 0; i < objKeys.length; i++) {
+    if (objValues.includes(objValues[i], i + 1)
+    || objKeys.includes(objKeys[i], i + 1)) {
```

Comment:
Why don't we just check if the result object already has such key?)

```suggestion
    if (trueRobot.hasOwnProperty(objValues[i])) {
```